game/db.py

```diff
@@ -330,7 +331,7 @@
 
 
 def upgrade_to_supercarrier(unit: Type[ShipType], name: str) -> Type[ShipType]:
-    if unit == Stennis:
+    if unit == Stennis or unit == VINSON:
```

Comment:
This is impossible and just future-proofing, right? I don't understand how this relates to the PR.

Reply:
Indeed, future-proofing.

Reply:
Drop all this from this PR and add it to that one IMO. We don't know that this is the structure the other PR will end up with, and right now it's just dead code that confuses the reader because it seems like an impossible case.